2021-09-01 22:09:12 database: opening /Users/tanjimhossain/Bytes/poc-wormhole/fromscratch1/.metals/metals
org.h2.message.DbException: Exception opening port "H2 TCP Server (tcp://localhost:50114)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-200]
	at org.h2.message.DbException.convert(DbException.java:335)
	at org.h2.engine.Database.startServer(Database.java:881)
	at org.h2.engine.Database.open(Database.java:658)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:268)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:232)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:55)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:68)
	at org.flywaydb.core.Flyway.execute(Flyway.java:510)
	at org.flywaydb.core.Flyway.clean(Flyway.java:361)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:123)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:111)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:105)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:54)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:37)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:44)
	at scala.meta.internal.metals.Tables.cancel(Tables.scala:40)
	at scala.meta.internal.metals.Cancelable$.$anonfun$cancelAll$1(Cancelable.scala:29)
	at scala.collection.immutable.List.foreach(List.scala:431)
	at scala.meta.internal.metals.Cancelable$.cancelAll(Cancelable.scala:28)
	at scala.meta.internal.metals.MutableCancelable.cancel(MutableCancelable.scala:25)
	at scala.meta.internal.metals.MetalsLanguageServer.cancel(MetalsLanguageServer.scala:116)
	at scala.meta.internal.metals.MetalsLanguageServer.cancelAll(MetalsLanguageServer.scala:128)
	at scala.meta.metals.Main$.main(Main.scala:75)
	at scala.meta.metals.Main.main(Main.scala)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Exception opening port "H2 TCP Server (tcp://localhost:50114)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:622)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.tools.Server.start(Server.java:530)
	at org.h2.engine.Database.startServer(Database.java:879)
	... 33 more
2021-09-01 22:09:31 database: opening /Users/tanjimhossain/Bytes/poc-wormhole/fromscratch1/.metals/metals
org.h2.message.DbException: Exception opening port "H2 TCP Server (tcp://localhost:50361)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-200]
	at org.h2.message.DbException.convert(DbException.java:335)
	at org.h2.engine.Database.startServer(Database.java:881)
	at org.h2.engine.Database.open(Database.java:658)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:268)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:232)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:55)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:68)
	at org.flywaydb.core.Flyway.execute(Flyway.java:510)
	at org.flywaydb.core.Flyway.migrate(Flyway.java:170)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:119)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:111)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:105)
	at scala.meta.internal.metals.Tables.tryNoAutoServer(Tables.scala:71)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:65)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:37)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:44)
	at scala.meta.internal.metals.Tables.cancel(Tables.scala:40)
	at scala.meta.internal.metals.Cancelable$.$anonfun$cancelAll$1(Cancelable.scala:29)
	at scala.collection.immutable.List.foreach(List.scala:431)
	at scala.meta.internal.metals.Cancelable$.cancelAll(Cancelable.scala:28)
	at scala.meta.internal.metals.MutableCancelable.cancel(MutableCancelable.scala:25)
	at scala.meta.internal.metals.MetalsLanguageServer.cancel(MetalsLanguageServer.scala:116)
	at scala.meta.internal.metals.MetalsLanguageServer.cancelAll(MetalsLanguageServer.scala:128)
	at scala.meta.metals.Main$.main(Main.scala:75)
	at scala.meta.metals.Main.main(Main.scala)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Exception opening port "H2 TCP Server (tcp://localhost:50361)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:622)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.tools.Server.start(Server.java:530)
	at org.h2.engine.Database.startServer(Database.java:879)
	... 34 more
2021-09-01 22:12:44 database: opening /Users/tanjimhossain/Bytes/poc-wormhole/fromscratch1/.metals/metals
org.h2.message.DbException: Exception opening port "H2 TCP Server (tcp://localhost:50721)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-200]
	at org.h2.message.DbException.convert(DbException.java:335)
	at org.h2.engine.Database.startServer(Database.java:881)
	at org.h2.engine.Database.open(Database.java:658)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:268)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:232)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:55)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:68)
	at org.flywaydb.core.Flyway.execute(Flyway.java:510)
	at org.flywaydb.core.Flyway.migrate(Flyway.java:170)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:119)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:111)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:105)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:54)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:37)
	at scala.meta.internal.metals.MetalsLanguageServer.initialized(MetalsLanguageServer.scala:879)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.notify(GenericEndpoint.java:152)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleNotification(RemoteEndpoint.java:220)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:187)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Exception opening port "H2 TCP Server (tcp://localhost:50721)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:622)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.tools.Server.start(Server.java:530)
	at org.h2.engine.Database.startServer(Database.java:879)
	... 40 more
2021-09-01 22:13:03 database: opening /Users/tanjimhossain/Bytes/poc-wormhole/fromscratch1/.metals/metals
org.h2.message.DbException: Exception opening port "H2 TCP Server (tcp://localhost:50967)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-200]
	at org.h2.message.DbException.convert(DbException.java:335)
	at org.h2.engine.Database.startServer(Database.java:881)
	at org.h2.engine.Database.open(Database.java:658)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:268)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:232)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:55)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:68)
	at org.flywaydb.core.Flyway.execute(Flyway.java:510)
	at org.flywaydb.core.Flyway.clean(Flyway.java:361)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:123)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:111)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:105)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:54)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:37)
	at scala.meta.internal.metals.MetalsLanguageServer.initialized(MetalsLanguageServer.scala:879)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.notify(GenericEndpoint.java:152)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleNotification(RemoteEndpoint.java:220)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:187)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Exception opening port "H2 TCP Server (tcp://localhost:50967)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:622)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.tools.Server.start(Server.java:530)
	at org.h2.engine.Database.startServer(Database.java:879)
	... 40 more
2021-09-01 22:13:22 database: opening /Users/tanjimhossain/Bytes/poc-wormhole/fromscratch1/.metals/metals
org.h2.message.DbException: Exception opening port "H2 TCP Server (tcp://localhost:51212)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-200]
	at org.h2.message.DbException.convert(DbException.java:335)
	at org.h2.engine.Database.startServer(Database.java:881)
	at org.h2.engine.Database.open(Database.java:658)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:268)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:232)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:55)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:68)
	at org.flywaydb.core.Flyway.execute(Flyway.java:510)
	at org.flywaydb.core.Flyway.migrate(Flyway.java:170)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:119)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:111)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:105)
	at scala.meta.internal.metals.Tables.tryNoAutoServer(Tables.scala:71)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:65)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:37)
	at scala.meta.internal.metals.MetalsLanguageServer.initialized(MetalsLanguageServer.scala:879)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.notify(GenericEndpoint.java:152)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleNotification(RemoteEndpoint.java:220)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:187)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Exception opening port "H2 TCP Server (tcp://localhost:51212)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:622)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.tools.Server.start(Server.java:530)
	at org.h2.engine.Database.startServer(Database.java:879)
	... 41 more
2021-09-01 22:13:41 database: opening /Users/tanjimhossain/Bytes/poc-wormhole/fromscratch1/.metals/metals
org.h2.message.DbException: Exception opening port "H2 TCP Server (tcp://localhost:51458)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-200]
	at org.h2.message.DbException.convert(DbException.java:335)
	at org.h2.engine.Database.startServer(Database.java:881)
	at org.h2.engine.Database.open(Database.java:658)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:268)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:232)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:55)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:68)
	at org.flywaydb.core.Flyway.execute(Flyway.java:510)
	at org.flywaydb.core.Flyway.clean(Flyway.java:361)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:123)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:111)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:105)
	at scala.meta.internal.metals.Tables.tryNoAutoServer(Tables.scala:71)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:65)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:37)
	at scala.meta.internal.metals.MetalsLanguageServer.initialized(MetalsLanguageServer.scala:879)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.notify(GenericEndpoint.java:152)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleNotification(RemoteEndpoint.java:220)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:187)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Exception opening port "H2 TCP Server (tcp://localhost:51458)" (port may be in use), cause: "timeout; please check your network configuration, specially the file /etc/hosts" [90061-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:622)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.tools.Server.start(Server.java:530)
	at org.h2.engine.Database.startServer(Database.java:879)
	... 41 more
2021-09-02 19:25:20 database: close
org.h2.message.DbException: IO Exception: "java.io.EOFException"; "/Users/tanjimhossain/Bytes/poc-wormhole/fromscratch1/.metals/metals.h2.db" [90031-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convertIOException(DbException.java:394)
	at org.h2.store.FileStore.readFully(FileStore.java:284)
	at org.h2.pagestore.PageStore.readPage(PageStore.java:1320)
	at org.h2.pagestore.PageStore.getPage(PageStore.java:744)
	at org.h2.pagestore.PageStreamTrunk$Iterator.next(PageStreamTrunk.java:252)
	at org.h2.pagestore.PageLog.free(PageLog.java:216)
	at org.h2.pagestore.PageStore.compact(PageStore.java:494)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1517)
	at org.h2.engine.Database.closeImpl(Database.java:1454)
	at org.h2.engine.Database.close(Database.java:1373)
	at org.h2.engine.OnExitDatabaseCloser.onShutdown(OnExitDatabaseCloser.java:85)
	at org.h2.engine.OnExitDatabaseCloser.run(OnExitDatabaseCloser.java:114)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "java.io.EOFException"; "/Users/tanjimhossain/Bytes/poc-wormhole/fromscratch1/.metals/metals.h2.db" [90031-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 13 more
Caused by: java.io.EOFException
	at org.h2.store.fs.FileUtils.readFully(FileUtils.java:361)
	at org.h2.store.FileStore.readFully(FileStore.java:282)
	... 10 more
